tests/CallStackTraitTest.php

```diff
@@ -56,7 +58,7 @@
 use PHPUnit_Framework_TestCase;
 use Grpc;
 
-class ApiCallableTest extends PHPUnit_Framework_TestCase
+class CallStackTraitTest extends PHPUnit_Framework_TestCase
```

Comment:
This may already be covered by the tests you have here, but it would be good to test using a "real" grpc transport object together with a mock grpc stub. I put real in quotes because it may not be that meaningful to call a transport "real" or "mock" if it is created specifically for testing. But what I mean is that we should test with a transport object that is as close as possible to the expected grpc transport class that will be generated, but with only the grpc stub mocked out.